src/runtime.js

```diff
@@ -74,6 +75,7 @@ export function installRuntimeServices(global) {
   installViewerService(global);
   installViewportService(global);
   installHistoryService(global);
+  installTimerService(global);
```

Comment:
It looks like timer should go before anything else. I see it's used by viewer and history services in particular.